test/README.md

```diff
@@ -1,11 +1,40 @@
 # Guide on testing
 
 - [Test with `TestManager` Module](#test-with-testmanager-module)
+    - [TestManager Overview](#Overview-About-testManager-Module)
     - [Compile](#compile)
-    - [Configure](#configure)
+    - [Configure](#Modify-the-configuration-files-accordingly)
     - [Run](#run)
     - [Verify](#verify)
 
+## Overview About testManager Module
+
+* testManager module basically designed to **mock** the **Cloud** messages which comes to **EdgeHub** for various operations on kubeedge node like, 
```

Comment:
module basically designed  -> module is designed

Reply:
we can rephrase like: testManager is a utility module to mimic the cloud and push messages for different kinds of actions that could happen from the cloud. Typical device , node and application lifecycle management functions are expected to be performed in the cloud and pushed to the edge node. These functions commonly encompass configurations related to  
     - Kubernetes Secrets and Configuration Maps.
     - Application deployment/sync.
     - Binding devices to edge nodes via memberships.
     - Syncing of different resources between cloud and edge ( like app status, device status etc)
     - Node sync, etc..